.github/CODEOWNERS

```diff
@@ -7,6 +7,8 @@
 # The @googleapis/yoshi-python is the default owner for changes in this repo
 *               @googleapis/yoshi-python
 
+# The @googleapis/cdpe-cloudai own the samples
+/samples/**/*.py                       @googleapis/cdpe-cloudai
 
 # The python-samples-owners team is the default owner for samples
 /samples/**/*.py                       @telpirion @sirtorry @googleapis/python-samples-owners
```

Comment:
Could you add @googleapis/cdpe-cloudai to this line instead? That way the sample owners will still be able to merge template changes. Thanks!